protos/sources/vega/api/v1/core.proto

```diff
@@ -343,3 +348,61 @@ message LastBlockHeightResponse {
   // the network chain id from which the block comes from
   string chain_id = 8;
 }
+
+// A request to retrieve the spam statistics of a party for the given epoch
+message GetSpamStatisticsRequest {
+  string party_id = 1;
+}
+
+// The statistics for a given spam policy
+message SpamStatistic {
+  // The current transaction count received from the party during this epoch for this policy
+  string count_for_epoch = 1;
+  // The maximum number of transactions allowed for this policy in an epoch
+  string max_for_epoch = 2;
+  // If blocked the timestamp when the party will be unblocked as RFC3339Nano
+  optional string banned_until = 4;
+}
+
+// The vote statistics for the voting spam policies
+// which are calculated as a ratio of the total votes
+// that have been rejected.
+message VoteSpamStatistic {
+  string proposal = 1;
+  // The current vote count received from the party for the given policy during this epoch
+  string count_for_epoch = 2;
+  // The maximum number of votes per proposal allowed in an epoch
+  string max_for_epoch = 3;
+  // If blocked the timestamp when the party will be unblocked as RFC3339Nano
+  optional string banned_until = 4;
+}
+
+// The PoW state for a given block
+message PoWBlockState {
+  uint64 block_height = 1;
+  uint64 transactions_seen = 2;
+  uint64 difficulty = 3;
+}
+
+// Proof of work statistics for a party
+message PoWStatistic {
+  // Block state for each block in scope for PoW calculation
+  repeated PoWBlockState block_states = 1;
+  // PoW banned until timestamp as RFC3339Nano
+  optional string banned_until = 2;
+}
+
+// The complete spam statistics captured for a given party
+message SpamStatistics {
+  SpamStatistic proposals = 1;
+  SpamStatistic delegations = 2;
+  SpamStatistic transfers = 3;
+  SpamStatistic node_announcements = 4;
+  repeated VoteSpamStatistic votes = 5;
```

Comment:
I think this is a bit awkward that vote here is a collection rather than an object with a collection because things like banned_until and effective_Min_Tokens (which I think is worth adding) are global for party votes rather than per proposal.